mediacapture-streams/MediaStreamTrackEvent-constructor.https.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<title>MediaStreamTrackEvent constructor</title>
+<link rel="help" href="https://w3c.github.io/mediacapture-main/#mediastreamtrackevent">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+test(function() {
+  assert_equals(MediaStreamTrackEvent.length, 2);
+  assert_throws(new TypeError, function() {
+    new MediaStreamTrackEvent("type");
+  });
+  assert_throws(new TypeError, function() {
+    new MediaStreamTrackEvent("type", null);
+  });
+  assert_throws(new TypeError, function() {
+    new MediaStreamTrackEvent("type", undefined);
+  });
+}, "The eventInitDict argument is required");
+
+test(function() {
+  assert_throws(new TypeError, function() {
+    new MediaStreamTrackEvent("type", {});
+  });
+  assert_throws(new TypeError, function() {
+    new MediaStreamTrackEvent("type", { track: null });
```

Comment:
This line is the main reason for writing the test, it revealed that Chromium has an IDL compiler bug that I'm looking into. I intend to import this test back to Blink once landed and it also reveals some things that are wrong in Gecko.